packages/manager/modules/bm-server-components/src/netboot/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,40 @@
+{
+  "server_configuration_netboot_title": "Changer le netboot",
+  "server_configuration_netboot_unavailable": "Information indisponible",
+  "server_configuration_netboot_title_help_rootdevice": "Root devices acceptés (N un nombre) :",
+  "server_configuration_netboot_title_help_rootdevice_list1": "/dev/hda[0 - N]",
+  "server_configuration_netboot_title_help_rootdevice_list2": "/dev/hdb[0 - N]",
+  "server_configuration_netboot_title_help_rootdevice_list3": "/dev/sda[0 - N]",
+  "server_configuration_netboot_title_help_rootdevice_list4": "/dev/sdb[0 - N]",
+  "server_configuration_netboot_title_help_rootdevice_list5": "/dev/md[0 - N]",
+  "server_configuration_netboot_title_help_rootdevice_list6": "/dev/rd/c0p0[0 - N]",
+  "server_configuration_netboot_title_help_rootdevice_list7": "/dev/ram0",
+  "server_configuration_netboot_current": "Netboot acutellement configuré sur votre serveur {{t0}} :",
+  "server_configuration_netboot_step1_question": "Que souhaitez-vous faire ?",
+  "server_configuration_netboot_option_harddisk": "Booter sur le disque dur",
+  "server_configuration_netboot_option_rescue": "Booter en mode rescue",
+  "server_configuration_netboot_option_network": "Booter en mode network",
+  "server_configuration_netboot_configuration": "Configuration :",
+  "server_configuration_netboot_rescue_choice": "Rescue disponible :",
+  "server_configuration_netboot_rescue_mail": "Recevoir les identifiants du mode sur l'adresse e-mail :",
+  "server_configuration_netboot_rescue_mail_invalid": "Invalide",
+  "server_configuration_netboot_kernel_choice": "Kernel disponible :",
+  "server_configuration_netboot_kernel_option": "Options prise en charges par ce kernel :",
+  "server_configuration_netboot_kernel_rootdevice": "Root device :",
+  "server_configuration_netboot_kernel_rootdevice_exemple": "exemple: voir aide",
```

Comment:
```suggestion
  "server_configuration_netboot_kernel_rootdevice_exemple": "exemple : voir aide",
```

Reply:
These translation messages are not new. I have just moved them from one place to another. No change will be made to the messages to avoid less impact.